config/config.example.toml

```
@@ -424,6 +424,11 @@ base_url = ""                        # Base url used for user specific redirects
 force_two_factor_auth = false        # Whether to force two factor authentication for all users
 force_cookies = true                 # Whether to use only cookies for JWT extraction and authentication
 
+[hubspot]
+enabled = false  # Whether to enable hubspot tracking
+request_url = ""    # Request url for hubspot tracking
+form_id=""       # Hubspot-specific form ID
```

(NIT)
Can we do this in all the configs.
```suggestion
form_id = ""       # Hubspot-specific form ID
```